content/1_General/Working_MDX.mdx

```diff
@@ -420,16 +430,40 @@ can be hovered over.
 ### Code Blocks and Code Snippets
 
 Code blocks are separated by three backticks, just like in normal markdown.
-Additionally, we have support for collapsible code snippets:
+Additionally, we have support for collapsible code snippets. An example for how
+to use them can be found below:
+
+**With title**
+
+```
+Before
+//BeginCodeSnip{Optional Code Snippet Title}
```

Comment:
i think the main point of confusion on how to use codesnips is caused by the fact that these codesnips are replaced by actual codesnips, so people don't see the syntax of putting them in a mdx file

idk if there's a way to escape them so they aren't replaced??

Reply:
![image](https://user-images.githubusercontent.com/43394822/124003910-11ffc600-d99d-11eb-8669-780680bffe1b.png)

yeah it looks like this on my end

Reply:
ohh nvm
i think the codesnips don't get replaced if they're not in a codeblock with language